gateway/sensor.go

```diff
@@ -188,16 +191,15 @@ func (g *Gateway) receivePackets() {
 			case 1:
 				// received a LORA packet
 				var payload []byte
-				for i := 0; i < numPackets; i++ {
-					if packet.size == 0 {
-						continue
-					}
-					// Convert packet to go byte array
-					for i := 0; i < int(packet.size); i++ {
-						payload = append(payload, byte(packet.payload[i]))
-					}
-					g.handlePacket(ctx, payload)
+				if packet.size == 0 {
+					continue
 				}
+				// Convert packet to go byte array
+				//nolint:intrange
```

Comment:
you can also add this to the `disable` section of your `etc/.golangci.yaml` file